### DISCUSSION TITLE
Data Types

### DISCUSSION THREAD

**Discussion by jsevic** [idaholab/moose#26056] [Q&A General] [answered]
Hello,

If I couple a variable _grad_V as coupledGradient in a kernel, which is a vector data type I think, and compute the product _grad_V * _grad_V, does this retain its vector identity or does it become a real value scalar?

Thank you.

**Comment by lindsayad** [comment, marked as answer]
it becomes a scalar

**Reply by jsevic** [reply]
Great it works thx!
